src/idl_gen_python.cpp

```diff
@@ -129,6 +129,11 @@ class PythonGenerator : public BaseGenerator {
     code += "(cls, buf, offset=0):";
     code += "\n";
     code += Indent + Indent;
+    code += "buf = flatbuffers.compat.memoryview_type(buf) ";
```

Comment:
Why is this conditional if the original use of it was unconditional? Are we expecting the condition to be false most of the time?

Reply:
That's because `memoryview(memoryview_instance)` creates new memoryview instance instead of returning the input:
```python
>>> buf=b'test_content'
>>> m=memoryview(buf)
>>> id(m)
140608505874560
>>> m2=memoryview(m)
>>> id(m2)
140608505873600
```
I didn't want to create `memoryview` of `memoryview` and I found a nested call from [here](https://github.com/google/flatbuffers/blob/c705ff1790391450abb6e592f589bc9b3b331906/src/idl_gen_python.cpp#L450). Indeed, this conditional if does not seem crucial though.